src/main/java/seedu/address/commons/core/LogsCenter.java

```diff
@@ -20,21 +20,31 @@
 public class LogsCenter {
     private static final int MAX_FILE_COUNT = 5;
     private static final int MAX_FILE_SIZE_IN_BYTES = (int) (Math.pow(2, 20) * 5); // 5MB
+    private static final String BASE_LOGGER_NAME = "seedu.address";
     private static final String LOG_FILE = "addressbook.log";
     private static Level currentLogLevel = Level.INFO;
     private static final Logger logger = LogsCenter.getLogger(LogsCenter.class);
     private static FileHandler fileHandler;
     private static ConsoleHandler consoleHandler;
+    private static final Logger baseLogger = Logger.getLogger(BASE_LOGGER_NAME);
 
     /**
-     * Initializes with a custom log level (specified in the {@code config} object)
-     * Loggers obtained *AFTER* this initialization will have their logging level changed<br>
-     * Logging levels for existing loggers will only be updated if the logger with the same name
-     * is requested again from the LogsCenter.
+     * Initializes with a custom log level (specified in the {@code config} object). All Loggers
+     * created using {@link #getLogger(String)} and {@link #getLogger(Class)} will have the same log
+     * level as the one specified in the {@code config} object.
```

Comment:
A slightly more concise version:

```suggestion
     * Initializes loggers with the log level specified in the {@code config} object. Applies to
     * all loggers created using {@link #getLogger(String)} and {@link #getLogger(Class)} methods.
```